source/css/sass/mixins/_lists.scss

```diff
@@ -0,0 +1,27 @@
+@import "spacing";
+@import "../locales";
+
+@mixin inline-list() {
+  list-style: none;
+  @include nospace("inline-start");
+}
+
+@mixin inline-list-item() {
```

Comment:
Did wonder about including this in `inline-list()` using `&__item`.

Reply:
I'n not too keen on partial selectors, because they can get missed during refactoring.